docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -0,0 +1,349 @@
+        # edpm firewall, change the allowed CIDR if needed
+        edpm_sshd_configure_firewall: true
+        edpm_sshd_allowed_ranges: ['192.168.122.0/24']
+        # SELinux module
+        edpm_selinux_mode: enforcing
+
+        # Do not attempt OVS major upgrades here
+        edpm_ovs_packages:
+        - openvswitch3.1
+EOF
+----
+
+. Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the `Networker` nodes before adoption. This configuration is stored in the `external_ids`` column in the `Open_vSwitch` table in the Open vSwitch database:
++
+----
+ovs-vsctl list Open .
+...
+external_ids        : {hostname=standalone.localdomain, ovn-bridge=br-int, ovn-bridge-mappings=<bridge_mappings>, ovn-chassis-mac-mappings="datacentre:1e:0a:bb:e6:7c:ad", ovn-encap-ip="172.19.0.100", ovn-encap-tos="0", ovn-encap-type=geneve, ovn-match-northd-version=False, ovn-monitor-all=True, ovn-ofctrl-wait-before-clear="8000", ovn-openflow-probe-interval="60", ovn-remote="tcp:ovsdbserver-sb.openstack.svc:6642", ovn-remote-probe-interval="60000", rundir="/var/run/openvswitch", system-id="2eec68e6-aa21-4c95-a868-31aeafc11736"}
+...
+----
+
++
+* Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
+
+. Optional: Enable `neutron-metadata` in the `OpenStackDataPlaneNodeSet` CR:
++
+[source,yaml]
+----
+oc patch openstackdataplanenodeset openstack-networker --type='json' --patch='[
+  {
+    "op": "add",
+    "path": "/spec/services/-",
+    "value": "neutron-metadata"
+  }]'
+----
+
+. Optional: Enable `neutron-dhcp` in the `OpenStackDataPlaneNodeSet` CR:
++
+[source,yaml]
+----
+oc patch openstackdataplanenodeset openstack-networker --type='json' --patch='[
+  {
+    "op": "add",
+    "path": "/spec/services/-",
+    "value": "neutron-dhcp"
+  }]'
+----
+
+. Run pre-adoption validation for `Networker` nodes. This step reuses the
+service that was previously created for `Compute` nodes:
+
+.. Create a `OpenStackDataPlaneDeployment` CR that runs the validation only:
++
+[source,yaml]
+----
+oc apply -f - <<EOF
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneDeployment
+metadata:
+  name: openstack-pre-adoption-networker
+spec:
+  nodeSets:
+  - openstack-networker
+  servicesOverride:
+  - pre-adoption-validation
+EOF
+----
++
+Wait for the validation to finish.
+
+.. Confirm that all the Ansible EE pods reach a `Completed` status:
++
+----
+# watching the pods
+watch oc get pod -l app=openstackansibleee
+----
++
+----
+# following the ansible logs with:
+oc logs -l app=openstackansibleee -f --max-log-requests 20
+----
+
+.. Wait for the deployment to reach the `Ready` status:
++
+----
+oc wait --for condition=Ready openstackdataplanedeployment/openstack-pre-adoption-networker --timeout=10m
+----
+
+. Deploy the `OpenStackDataPlaneDeployment` CR for `Networker` nodes:
```

Comment:
do we need new Deployment CR for that? What if we would add new NodeSet to the existing Deployment CR?

Reply:
Yeah, I guess I went with simpler separation of two topics (compute vs networker) though both steps should be aligned better. I think we may want to document both. There are benefits with a separate Deployment (failure domain reduction), but it's helpful to demonstrate that the whole deployment can be done in a single step too.

Reply:
I am adding a note to both sections of the document that you can either deploy separately, or reuse the same CR (plus a note that patching after deployment won't work.)